test/functional/services/inspector.ts

```diff
@@ -192,22 +192,29 @@ export class InspectorService extends FtrService {
   /**
    * Opens inspector view
    * @param viewId
+   *  This method was flaky. So we changed viewId to the substring of the dts which is visible
```

Comment:
How about simplifying to just "Inspector view id, for example 'Requests'". I think including history makes the API harder to understand. History is captured by git and github and should not be documented in API.